dist/tools/firmware_metadata/Makefile

```diff
@@ -0,0 +1,19 @@
+RIOTBASE := ../../..
+RIOT_INCLUDE = $(RIOTBASE)/sys/include
+SHA256_DIR := $(RIOTBASE)/sys/hashes
+SHA256_INCLUDE := $(RIOT_INCLUDE)/hashes
+METADATA_SRC := generate-metadata.c $(SHA256_DIR)/sha256.c
+METADATA_HDR := $(RIOT_INCLUDE)/hashes/sha256.h
+
+CFLAGS += -g -O3 -Wall -Wextra -pedantic -std=c99
+
+all: bin bin/generate-metadata
+
+bin:
+	mkdir bin
```

Comment:
maybe use `-p` to avoid messages like `mkdir: cannot create directory ‘bin’: File exists`